template/module/__openerp__.py

```diff
@@ -0,0 +1,15 @@
+# -*- encoding: utf-8 -*-
+{
+    "name": "Module name",
+    "summary": "Module summary",
+    "version": "0.1.0",
+    "author": "Odoo Community Association (OCA)",
+    "category": "Module category",
+    "depends": [
+        "base",
+    ],
```

Comment:
A little known key is `external_dependencies`. I would argue that adding it in the template would help newer devs who do not know about it yet.
```
    'external_dependencies': {
        'python': [],
        'bin': [],
    },
```